test/net/sourceforge/kolmafia/request/PlaceRequestTest.java

```diff
@@ -74,7 +79,75 @@ void itShouldSetTheToolbeltAsAFreePullInTTT() {
     assertEquals(
         -1, freePulls.indexOf(toolbelt), "toolbelt should not be in Free Pulls after TTT fades");
   }
-}
 
-// Generated with love by TestMe :) Please report issues and submit feature requests at:
-// http://weirddev.com/forum#!/testme
+  @Nested
+  class Speakeasy {
+    @BeforeEach
```

Comment:
Why are these @BeforeEach and @AfterEach classes here?
They were copied right from the top of the file - where they apply to all tests, even Nested ones.

That said, they are dubious. They are commented:
```
  // These need to be before and after each because leakage has been observed between tests
  // in this class.
```

Before this, there was a single test in this class.
The correct fix would have been to use Cleanups.

Make the BeforeAll do a single KoLCharacter.reset.
Make the BeforeEach do a single Preferences.reset